test/bandit/clock_test.exs

```diff
@@ -0,0 +1,17 @@
+defmodule ClockTest do
+  use ExUnit.Case, async: false
+  import ExUnit.CaptureLog
+
+  test "clock emits warning if ETS cache isn't available" do
+    Application.stop(:bandit)
```

Comment:
@mtrudel 
Is there a better way to check the warnings than stopping and starting the application?

Reply:
No, I think this is fine!